pennylane_lightning/lightning_qubit/lightning_qubit2.py

```diff
@@ -208,12 +232,13 @@ class LightningQubit2(Device):
     """
 
     name = "lightning.qubit2"
+    _CPP_BINARY_AVAILABLE = LQ_CPP_BINARY_AVAILABLE
 
     _device_options = ["rng", "c_dtype", "batch_obs", "mcmc", "kernel_name", "num_burnin"]
 
     def __init__(  # pylint: disable=too-many-arguments
         self,
-        wires=None,
+        wires=None,  # the number of wires are always needed for Lightning. Do we have a better way to get it
```

Comment:
We can just make `wires` mandatory for lightning qubit.

Reply:
I will make it mandatory.

Reply:
I'm rethinking that. I think it is better to query it from the tape. As we have now.